app/views/pages/start.html.erb

```diff
@@ -10,4 +10,17 @@
   <li>view and download performance reports</li>
 </ul>
 
-<%= govuk_button_link_to "Start now", dashboard_path %>
+
+<%= govuk_button_to(user_cis2_omniauth_authorize_path, class: "app-button--cis2 nhsuk-u-margin-bottom-4") do %>
+  <svg class="nhsuk-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 32" height="27" aria-hidden="true" focusable="false">
+    <path fill="currentColor" d="M80 0v32H0V0h80ZM69 2.2c-5.8 0-11.6 2-11.6 8.8 0 7.4 10.2 5.8 10.2 10 0 2.6-3.4 3-5.6 3-2.2 0-5-.6-6.4-1.4L54 28c2.2.8 5.4 1.4 8 1.4 6.2 0 12.8-1.8 12.8-9 0-7.8-10.2-6.6-10.2-10.2 0-2.2 2.2-2.6 5-2.6 2.6 0 4.4.6 5.8 1.2L77 3.4c-1.8-.8-4.8-1.2-8-1.2ZM16.6 3H7.8L2.2 29h6.6l3.6-18h.2L18 29h8.6l5.6-26h-6.6L22 21h-.2L16.6 3Zm25.2 0h-7.2l-5.2 26h6.8l2.4-11.2h8.2L44.6 29h7L57 3h-7l-2.2 9.8h-8l2-9.8Z"></path>
+  </svg>
+
+  <span>Care Identity</span>
+<% end %>
+
+<p class="nhsuk-body">
+  <a class="nhsuk-link" href="https://manage-care-identities.care-identity-service2.nhs.uk/#/self-service/unlock-authenticator">
```

Comment:
This goes to a 404 page for me, is the URL incorrect or is it not live yet?

Reply:
I'm getting this page, which is an error but not a 404. tbh I'm not sure exactly how this is meant to work, but I thought we could 🥾🥫 until we better understood this stuff.

![image](https://github.com/nhsuk/manage-vaccinations-in-schools/assets/15608/503a1642-f804-4c63-a74c-8cd256821e94)

Reply:
Turns out this is a country thing, looks like that page only works in the UK, it worked when I switched on my VPN. I've attached what it looks like outside the UK.

<img width="295" alt="Screenshot 2024-06-27 at 13 16 47" src="https://github.com/nhsuk/manage-vaccinations-in-schools/assets/510498/a6831aa3-e772-421a-a42e-f13c3cdb2897">

Reply:
Ah, interesting. The dev CIS2 worked when I was out of the country recently, and I was wondering if production was going to be region-restricted. Glad to see it is.